aten/src/ATen/native/TensorFactories.cpp

```diff
@@ -577,6 +596,17 @@ Tensor randint_like(
   return result.random_(low, high, c10::nullopt);
```

Comment:
Make is similar to `randint` and `randint_out` above: reuse new `randint_like` here
```suggestion
  return native::randint_like(self, low, high, c10::nullopt, options, optional_memory_format);
```